apps/functions/document-unpublish/host.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
So this isn't something we do much of right now, but we can probably group the unpublish-doocument and publish-document functions together if you're comfortable making this change? It makes sense that they'd live and change together, and they share some common code.

Reply:
For context, just looking to make the amount of function apps we have as small as possible - since each new function app comes at a cost of a duplicated runtime and extra steps in the release pipeline. I think in future we might squash this number down even more, but do you think it makes sense for this change to do it right now?